src/monitor.rs

```diff
@@ -1,89 +1,64 @@
 //! Types useful for interacting with a user's monitors.
+use std::fmt;
 use std::num::{NonZeroU16, NonZeroU32};
 
 use crate::dpi::{PhysicalPosition, PhysicalSize};
 use crate::platform_impl;
 
-/// A handle to a fullscreen video mode of a specific monitor.
+/// Describes a fullscreen video mode of a monitor.
 ///
-/// This can be acquired with [`MonitorHandle::video_modes`].
-#[derive(Clone, PartialEq, Eq, Hash)]
-pub struct VideoModeHandle {
-    pub(crate) video_mode: platform_impl::VideoModeHandle,
+/// Can be acquired with [`MonitorHandle::video_modes`].
```

Comment:
Nit: "acquired" sounds like a method of ownership. Perhaps replace with "retrieved"?

```suggestion
/// Can be retrieved with [`MonitorHandle::video_modes()`].
```